### ISSUE TITLE
[BUG] Generated Update URI is Incorrect

### ISSUE THREAD

**Issue by riferrei** [opensearch-project/opensearch-go#145]
### What is the bug?
The generated HTTP request for the `opensearchapi.UpdateRequest` command is incorrect. The position of the documentID parameter should be the last one, but instead, it is coming before the `_update` argument. In other words:

Instead of:

`POST /sample-index/_update/1`

It is doing:

`POST /sample-index/1/_update`

### How can one reproduce the bug?
Here is the code that can be used to reproduce the bug:

```
	updateRequest := opensearchapi.UpdateRequest{
		Index:      backendIndex,
		DocumentID: documentID,
		Body:       bodyReader,
	}
	updateResponse, err := updateRequest.Do(ctx, backendClient)
```

### What is the expected behavior?
The generated URL for the update API should be: `POST /sample-index/_update/1`

### What is your host/environment?
MacOS, Monterey, ARM64

### Do you have any screenshots?
N/A

### Do you have any additional context?
N/A

**Comment by iquirino** [comment]
+1

**Comment by VachaShah** [comment]
This issue is fixed in #134 and released in version 2.0.1 - https://pkg.go.dev/github.com/opensearch-project/opensearch-go/v2.

**Comment by riferrei** [comment]
Just tested with v2.0.1 and it works as expected. Closing the issue. Thanks!
